### DISCUSSION TITLE
Is audio conversion function available through UI?

### DISCUSSION THREAD

**Discussion by smishberma** [kannagi0303/yt-dlp-gui#75] [Q&A] [answered]
In the changelog for version 2023.03.08?, it says:

>  [Add] Download Audio Can Convert to OGG, AAC, MP3...

I've been looking for this option in the UI but couldn't find any. Sorry, I'm a newb.

**Comment by kannagi0303** [comment, marked as answer]
Hello, on the right side of the Audio section in Formats, there is a downward arrow button. Clicking on it will bring up a save dialog box. Basically, you just need to change the file extension to the format you want. However, since I have only done simple tests, it may not be possible to save in the exact format you want. If this happens, please report it to me.
